Sources/KIF/Classes/KIFUIViewTestActor.m

```diff
@@ -155,6 +156,26 @@ - (instancetype)usingFirstResponder;
     return [self usingPredicate:predicate];
 }
 
+- (instancetype)usingCustomActionWithName:(NSString *)name
+{
+    NSPredicate *predicate = [NSPredicate predicateWithBlock:^BOOL(id evaluatedObject, NSDictionary *bindings) {
+        NSArray *actions = [evaluatedObject accessibilityCustomActions];
+        for (UIAccessibilityCustomAction *action in actions) {
+            NSString *actionName = [action name];
+            if ([actionName isKindOfClass:[NSAttributedString class]]) {
+                actionName = [(NSAttributedString *)actionName string];
+            }
+
+            if ([actionName isEqualToString: name]) {
+                return true;
+            }
+        }
```

Comment:
NIT: Up to you, but maybe worth a helper method for getting the specified action by name? Its identical here and below and might make these methods slightly simpler.